### DISCUSSION TITLE
LEARN

### DISCUSSION THREAD

**Discussion by pipoherreca** [community/community#35133] [Other Feature Feedback, Questions, & Ideas]
where i can find how learn about git an github..?.thanks

**Comment by radar-blip** [comment]
Here: [Git and GitHub learning resources](https://docs.github.com/en/get-started/quickstart/git-and-github-learning-resources)

**Comment by SilasRodrigues19** [comment]
@pipoherreca To learn more about GitHub, I recommend u read this [Quickstart](https://docs.github.com/en/get-started/quickstart/hello-world).
To learn about git commands I recommend this link: [Git commands](https://gist.github.com/scootcho/7bda4a469f9a0591cd8b) and there's also this really cool website that shows you the right commands you need to use: [Git Explorer](https://gitexplorer.com/)

**Comment by Manish5121** [comment]
i would recommend [this](https://youtu.be/apGV9Kg7ics) video
also try contributing to some very basic repositories
